packages/schema-poller/src/schemaPoller.ts

```diff
@@ -76,11 +79,15 @@ export class Neo4jSchemaPoller {
     }
 
     this.reconnectionTimeout = setTimeout(() => {
-      void this.persistentConnect(url, credentials, config);
+      void this.persistentConnect(url, credentials, config, database);
     }, 30000);
   }
 
   disconnect() {
+    if (this.connection) {
```

Comment:
not strictly a required change, but it is handy to see in the console when we disconnect